website/projects/scimaps/src/app/shared/news-item/news-item.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { Shallow } from 'shallow-render';
+
+import { NewsItemComponent } from './news-item.component';
+import { NewsItemModule } from './news-item.module';
+import { NewsItem } from './news-item.model';
+
+
+const testItem: NewsItem = {
+  title: 'test title',
+  date: 'test date',
+  source: 'test source',
+  institution: 'test institution',
+  thumbnail: 'test thumbnail',
+  pdfLink: 'test link'
+};
+
+describe('NewsItemComponent', () => {
+  let shallow: Shallow<NewsItemComponent>;
+
```

Comment:
A helper function for reducing duplication:
```typescript
function itHasElementContent(selector: string, content: string): void {
  it(`has the correct content in ${selector}`, async () => {
    const { find } = await shallow.render({ bind: { item: testItem } });
    const el = find(selector).nativeElement as Element;
    expect(el.innerHTML).toContain(content);
  }
}
```
Then this can simply be used as `itHasElementContent('.date', 'test date')`